docs/integrations/data-ingestion/clickpipes/mongodb/quickstart.md

```diff
@@ -266,11 +266,11 @@ LIMIT 10;
 
 ### Refreshable materialized view {#refreshable-materialized-view}
 
-You can also create [Refreshable Materialized Views](https://clickhouse.com/docs/materialized-view/refreshable-materialized-view), which enable you to schedule query execution for deduplicating rows and storing the results in a flattened destination table. With each scheduled refresh, the destination table is replaced with the latest query results.
+If your table size is small, you can create [Refreshable Materialized Views](https://clickhouse.com/docs/materialized-view/refreshable-materialized-view), which enable you to schedule query execution for deduplicating rows and storing the results in a flattened destination table. With each scheduled refresh, the destination table is replaced with the latest query results.
```

Comment:
If I was the user of this documentation, I would ask myself the question "what does small mean"? 1MB? 1GB? 1TB?